static/usr/lib/core/fix-hctosys

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# This script is a workaround for the issue that when an rtc module is inserted
+# the clock gets reset to whatever time the RTC has regardless of system time.
+# See https://github.com/snapcore/core20/pull/136 for more details
+
+
+# Debugging udev scripts is hard, if debugging is needed uncomment the following
+# lines and rebuild core with that.
+#set -x
+#if [ -e /run/mnt/ubuntu-seed ]; then
+#    LOG=/run/mnt/ubuntu-seed/fix-hctosys.log
+#else
+#    LOG=/run/fix-hctosys.log
+#fi
+#exec >> "$LOG"
+#exec 2>&1
+#printf "%s\n" "Starting $(date)"
+
+TIMESYNC_CLOCK=/var/lib/systemd/timesync/clock
+CLOCK_EPOCH=/var/lib/clock-epoch
+SELF=$(readlink -f "$0")
```

Comment:
You could get rid of `SELF` by using `#!/bin/bash` and "stating" `${BASH_SOURCE}`. Just a suggestion. Not important.